### DISCUSSION TITLE
Please open an issure or a new discussion, if you have any problem.

### DISCUSSION THREAD

**Discussion by Love4yzp** [Seeed-Solution/SenseCAP_Indicator_ESP32#22] [Announcements]
If you‘ve encountered a problem, please open an issure or take a new discussion.

**Comment by meberli** [comment]
can you please add the squareline projects to indicator_ha and other examples as well?

**Reply by Love4yzp** [reply]
It already had been there: https://github.com/Seeed-Solution/SenseCAP_Indicator_ESP32/tree/main/examples/squareline_demo

**Reply by meberli** [reply]
I can't find the squareline project for the  indicator_ha example there - only for the squareline example..

**Comment by meberli** [comment]
squareline projects are not included in the examples except in the squareline_demo example ..

**Comment by ugurakas** [comment]
I have your products and working on espressif but ı can't use espressif example or any other examples this products only work with your own examples ı can't run any other library on your products, How can ı fix it?

**Reply by Love4yzp** [reply]
do you mean you want to use the example that using esp-idf, but not in this project? We implement a bsp for sensecap indicator to let you quickly utilize the source hardware provided.

**Reply by ugurakas** [reply]
For example ı want to make bt scanner or any other things to any device also ı have esp32c3 and ı try to implement esp_bt library but ı can't implement any other library after set up your tools
